command_list.md

```diff
@@ -0,0 +1,116 @@
+# Account
+```
+rucio account list
+rucio account add
+rucio account show
+rucio account remove
+rucio account history
```

Comment:
What is `account history`?

Reply:
It's "list_account_usage". Renamed to history to match the other usage<->history mappings.

Reply:
In my mind, `history` doesn’t feel appropriate because it’s a current snapshot. I’m curious whether we can somehow fit this information under `account limits`. I’ll try to think more on it.

Reply:
```console
$ rucio list-account-usage dichrist
+------------------------------------------+----------+------------+--------------+
| RSE                                      | USAGE    | LIMIT      | QUOTA LEFT   |
|------------------------------------------+----------+------------+--------------|
| AGLT2_SCRATCHDISK                        | 0.000 B  | 50.000 TB  | 50.000 TB    |
| BEIJING-LCG2-EOS_SCRATCHDISK             | 0.000 B  | 7.500 TB   | 7.500 TB     |
| BNL-OSG2_SCRATCHDISK                     | 0.000 B  | 50.000 TB  | 50.000 TB    |
| BNLHPC_SCRATCHDISK                       | 0.000 B  | 15.000 TB  | 15.000 TB    |
| CA-SFU-T2_SCRATCHDISK                    | 0.000 B  | 48.000 TB  | 48.000 TB    |
| CA-VICTORIA-WESTGRID-T2_SCRATCHDISK      | 0.000 B  | 24.000 TB  | 24.000 TB    |
| CA-WATERLOO-T2_SCRATCHDISK               | 0.000 B  | 15.000 TB  | 15.000 TB    |
| CERN-PROD_SCRATCHDISK                    | 0.000 B  | 50.000 TB  | 50.000 TB    |
| CSCS-LCG2_SCRATCHDISK                    | 0.000 B  | 19.200 TB  | 19.200 TB    |
| CYFRONET-LCG2-EOS_SCRATCHDISK            | 0.000 B  | 30.000 TB  | 30.000 TB    |
| DESY-HH_LOCALGROUPDISK                   | 0.000 B  | 10.000 TB  | 10.000 TB    |
| DESY-HH_SCRATCHDISK                      | 0.000 B  | 50.000 TB  | 50.000 TB    |
| DESY-ZN_LOCALGROUPDISK                   | 0.000 B  | 10.000 TB  | 10.000 TB    |
| DESY-ZN_SCRATCHDISK                      | 0.000 B  | 15.000 TB  | 15.000 TB    |
| EELA-UTFSM_SCRATCHDISK                   | 0.000 B  | 7.680 TB   | 7.680 TB     |
| FMPHI-UNIBA_SCRATCHDISK                  | 0.000 B  | 16.493 TB  | 16.493 TB    |
| FZK-LCG2_SCRATCHDISK                     | 0.000 B  | 50.000 TB  | 50.000 TB    |
| GOEGRID_LOCALGROUPDISK                   | 0.000 B  | 10.000 TB  | 10.000 TB    |
| GOEGRID_SCRATCHDISK                      | 0.000 B  | 45.493 TB  | 45.493 TB    |
| GRIF_SCRATCHDISK                         | 0.000 B  | 50.000 TB  | 50.000 TB    |
| HK-LCG2_SCRATCHDISK                      | 0.000 B  | 32.985 TB  | 32.985 TB    |
| IEPSAS-KOSICE_SCRATCHDISK                | 0.000 B  | 26.543 TB  | 26.543 TB    |
| IFAE_SCRATCHDISK                         | 0.000 B  | 50.000 TB  | 50.000 TB    |
| IFIC-LCG2_SCRATCHDISK                    | 0.000 B  | 50.000 TB  | 50.000 TB    |
| IN2P3-CC_SCRATCHDISK                     | 0.000 B  | 49.808 TB  | 49.808 TB    |
| IN2P3-CPPM_SCRATCHDISK                   | 0.000 B  | 9.664 TB   | 9.664 TB     |
| IN2P3-LAPP-DCACHE_SCRATCHDISK            | 0.000 B  | 15.000 TB  | 15.000 TB    |
| IN2P3-LPC_SCRATCHDISK                    | 0.000 B  | 15.833 TB  | 15.833 TB    |
| INFN-COSENZA_SCRATCHDISK                 | 0.000 B  | 15.000 TB  | 15.000 TB    |
| INFN-FRASCATI_SCRATCHDISK                | 0.000 B  | 13.194 TB  | 13.194 TB    |
| INFN-GENOVA_SCRATCHDISK                  | 0.000 B  | 2.639 TB   | 2.639 TB     |
| INFN-LECCE_SCRATCHDISK                   | 0.000 B  | 5.236 TB   | 5.236 TB     |
| INFN-MILANO_SCRATCHDISK                  | 0.000 B  | 7.200 TB   | 7.200 TB     |
| INFN-NAPOLI-ATLAS_SCRATCHDISK            | 0.000 B  | 23.090 TB  | 23.090 TB    |
| INFN-ROMA1_SCRATCHDISK                   | 0.000 B  | 26.388 TB  | 26.388 TB    |
| INFN-ROMA3_SCRATCHDISK                   | 0.000 B  | 1.812 TB   | 1.812 TB     |
| INFN-T1_SCRATCHDISK                      | 0.000 B  | 50.000 TB  | 50.000 TB    |
| JINR-LCG2_SCRATCHDISK                    | 0.000 B  | 15.036 TB  | 15.036 TB    |
| LRZ-LMU_LOCALGROUPDISK                   | 0.000 B  | 381.065 TB | 381.065 TB   |
| LRZ-LMU_SCRATCHDISK                      | 0.000 B  | 31.111 TB  | 31.111 TB    |
| MAINZ_LOCALGROUPDISK                     | 0.000 B  | 950.000 TB | 950.000 TB   |
| MPPMU_LOCALGROUPDISK                     | 0.000 B  | 380.000 TB | 380.000 TB   |
| MPPMU_SCRATCHDISK                        | 0.000 B  | 30.000 TB  | 30.000 TB    |
| MWT2_UC_SCRATCHDISK                      | 0.000 B  | 50.000 TB  | 50.000 TB    |
| NCG-INGRID-PT_SCRATCHDISK                | 0.000 B  | 3.000 TB   | 3.000 TB     |
| NDGF-T1_SCRATCHDISK                      | 4.108 GB | 50.000 TB  | 49.996 TB    |
| NET2_SCRATCHDISK                         | 0.000 B  | 21.000 TB  | 21.000 TB    |
| NIKHEF_SCRATCHDISK                       | 0.000 B  | 16.685 TB  | 16.685 TB    |
| OU_OSCER_ATLAS_SCRATCHDISK               | 0.000 B  | 15.000 TB  | 15.000 TB    |
| PIC_SCRATCHDISK                          | 0.000 B  | 50.000 TB  | 50.000 TB    |
| PRAGUELCG2_SCRATCHDISK                   | 0.000 B  | 32.985 TB  | 32.985 TB    |
| RAL-LCG2-ECHO_SCRATCHDISK                | 0.000 B  | 50.000 TB  | 50.000 TB    |
| RO-07-NIPNE_SCRATCHDISK                  | 0.000 B  | 32.985 TB  | 32.985 TB    |
| RRC-KI-T1_SCRATCHDISK                    | 0.000 B  | 28.635 TB  | 28.635 TB    |
| RU-PROTVINO-IHEP_SCRATCHDISK             | 0.000 B  | 15.000 TB  | 15.000 TB    |
| SAMPA_SCRATCHDISK                        | 0.000 B  | 46.800 TB  | 46.800 TB    |
| SARA-MATRIX_SCRATCHDISK                  | 4.108 GB | 32.975 TB  | 32.971 TB    |
| SLACXRD_SCRATCHDISK                      | 0.000 B  | 30.000 TB  | 30.000 TB    |
| SWT2_CPB_SCRATCHDISK                     | 0.000 B  | 50.000 TB  | 50.000 TB    |
| TECHNION-HEP_SCRATCHDISK                 | 0.000 B  | 7.500 TB   | 7.500 TB     |
| TOKYO-LCG2_SCRATCHDISK                   | 0.000 B  | 50.000 TB  | 50.000 TB    |
| TR-10-ULAKBIM_SCRATCHDISK                | 0.000 B  | 9.896 TB   | 9.896 TB     |
| TRIUMF-LCG2_SCRATCHDISK                  | 0.000 B  | 50.000 TB  | 50.000 TB    |
| TW-LCG2_SCRATCHDISK                      | 0.000 B  | 30.221 TB  | 30.221 TB    |
| UAM-LCG2_SCRATCHDISK                     | 0.000 B  | 7.500 TB   | 7.500 TB     |
| UKI-LT2-QMUL_SCRATCHDISK                 | 0.000 B  | 50.000 TB  | 50.000 TB    |
| UKI-NORTHGRID-LANCS-HEP-CEPH_SCRATCHDISK | 0.000 B  | 50.000 TB  | 50.000 TB    |
| UKI-NORTHGRID-LIV-HEP_SCRATCHDISK        | 0.000 B  | 13.194 TB  | 13.194 TB    |
| UKI-NORTHGRID-MAN-HEP-CEPH_SCRATCHDISK   | 0.000 B  | 49.478 TB  | 49.478 TB    |
| UKI-NORTHGRID-MAN-HEP_SCRATCHDISK        | 0.000 B  | 49.478 TB  | 49.478 TB    |
| UKI-SCOTGRID-GLASGOW-CEPH_SCRATCHDISK    | 0.000 B  | 50.000 TB  | 50.000 TB    |
| UKI-SOUTHGRID-RALPP_SCRATCHDISK          | 0.000 B  | 7.500 TB   | 7.500 TB     |
| UNI-BONN_LOCALGROUPDISK                  | 0.000 B  | 10.000 TB  | 10.000 TB    |
| UNI-FREIBURG_LOCALGROUPDISK              | 0.000 B  | 475.000 TB | 475.000 TB   |
| UNI-FREIBURG_SCRATCHDISK                 | 0.000 B  | 18.000 TB  | 18.000 TB    |
| WUPPERTALPROD_LOCALGROUPDISK             | 0.000 B  | 1.000 TB   | 1.000 TB     |
| WUPPERTALPROD_SCRATCHDISK                | 0.000 B  | 19.792 TB  | 19.792 TB    |
+------------------------------------------+----------+------------+--------------+
+------------------+----------+------------+--------------+
| RSE EXPRESSION   | USAGE    | LIMIT      | QUOTA LEFT   |
|------------------+----------+------------+--------------|
| type=SCRATCHDISK | 8.217 GB | 200.000 TB | 199.992 TB   |
+------------------+----------+------------+--------------+
$ rucio list-account-limits dichrist
+------------------------------------------+------------+
| RSE                                      | LIMIT      |
|------------------------------------------+------------|
| AGLT2_SCRATCHDISK                        | 50.000 TB  |
| BEIJING-LCG2-EOS_SCRATCHDISK             | 7.500 TB   |
| BNL-OSG2_SCRATCHDISK                     | 50.000 TB  |
| BNLHPC_SCRATCHDISK                       | 15.000 TB  |
| CA-SFU-T2_SCRATCHDISK                    | 48.000 TB  |
| CA-VICTORIA-WESTGRID-T2_SCRATCHDISK      | 24.000 TB  |
| CA-WATERLOO-T2_SCRATCHDISK               | 15.000 TB  |
| CERN-PROD_SCRATCHDISK                    | 50.000 TB  |
| CSCS-LCG2_SCRATCHDISK                    | 19.200 TB  |
| CYFRONET-LCG2-EOS_SCRATCHDISK            | 30.000 TB  |
| DESY-HH_LOCALGROUPDISK                   | 10.000 TB  |
| DESY-HH_SCRATCHDISK                      | 50.000 TB  |
| DESY-ZN_LOCALGROUPDISK                   | 10.000 TB  |
| DESY-ZN_SCRATCHDISK                      | 15.000 TB  |
| EELA-UTFSM_SCRATCHDISK                   | 7.680 TB   |
| FMPHI-UNIBA_SCRATCHDISK                  | 16.493 TB  |
| FZK-LCG2_SCRATCHDISK                     | 50.000 TB  |
| GOEGRID_LOCALGROUPDISK                   | 10.000 TB  |
| GOEGRID_SCRATCHDISK                      | 45.493 TB  |
| GRIF_SCRATCHDISK                         | 50.000 TB  |
| HK-LCG2_SCRATCHDISK                      | 32.985 TB  |
| IEPSAS-KOSICE_SCRATCHDISK                | 26.543 TB  |
| IFAE_SCRATCHDISK                         | 50.000 TB  |
| IFIC-LCG2_SCRATCHDISK                    | 50.000 TB  |
| IN2P3-CC_SCRATCHDISK                     | 49.808 TB  |
| IN2P3-CPPM_SCRATCHDISK                   | 9.664 TB   |
| IN2P3-LAPP-DCACHE_SCRATCHDISK            | 15.000 TB  |
| IN2P3-LPC_SCRATCHDISK                    | 15.833 TB  |
| INFN-COSENZA_SCRATCHDISK                 | 15.000 TB  |
| INFN-FRASCATI_SCRATCHDISK                | 13.194 TB  |
| INFN-GENOVA_SCRATCHDISK                  | 2.639 TB   |
| INFN-LECCE_SCRATCHDISK                   | 5.236 TB   |
| INFN-MILANO_SCRATCHDISK                  | 7.200 TB   |
| INFN-NAPOLI-ATLAS_SCRATCHDISK            | 23.090 TB  |
| INFN-ROMA1_SCRATCHDISK                   | 26.388 TB  |
| INFN-ROMA3_SCRATCHDISK                   | 1.812 TB   |
| INFN-T1_SCRATCHDISK                      | 50.000 TB  |
| JINR-LCG2_SCRATCHDISK                    | 15.036 TB  |
| LRZ-LMU_LOCALGROUPDISK                   | 381.065 TB |
| LRZ-LMU_SCRATCHDISK                      | 31.111 TB  |
| MAINZ_LOCALGROUPDISK                     | 950.000 TB |
| MPPMU_LOCALGROUPDISK                     | 380.000 TB |
| MPPMU_SCRATCHDISK                        | 30.000 TB  |
| MWT2_UC_SCRATCHDISK                      | 50.000 TB  |
| NCG-INGRID-PT_SCRATCHDISK                | 3.000 TB   |
| NDGF-T1_SCRATCHDISK                      | 50.000 TB  |
| NET2_SCRATCHDISK                         | 21.000 TB  |
| NIKHEF_SCRATCHDISK                       | 16.685 TB  |
| OU_OSCER_ATLAS_SCRATCHDISK               | 15.000 TB  |
| PIC_SCRATCHDISK                          | 50.000 TB  |
| PRAGUELCG2_SCRATCHDISK                   | 32.985 TB  |
| RAL-LCG2-ECHO_SCRATCHDISK                | 50.000 TB  |
| RO-07-NIPNE_SCRATCHDISK                  | 32.985 TB  |
| RRC-KI-T1_SCRATCHDISK                    | 28.635 TB  |
| RU-PROTVINO-IHEP_SCRATCHDISK             | 15.000 TB  |
| SAMPA_SCRATCHDISK                        | 46.800 TB  |
| SARA-MATRIX_SCRATCHDISK                  | 32.975 TB  |
| SLACXRD_SCRATCHDISK                      | 30.000 TB  |
| SWT2_CPB_SCRATCHDISK                     | 50.000 TB  |
| TECHNION-HEP_SCRATCHDISK                 | 7.500 TB   |
| TOKYO-LCG2_SCRATCHDISK                   | 50.000 TB  |
| TR-10-ULAKBIM_SCRATCHDISK                | 9.896 TB   |
| TRIUMF-LCG2_SCRATCHDISK                  | 50.000 TB  |
| TW-LCG2_SCRATCHDISK                      | 30.221 TB  |
| UAM-LCG2_SCRATCHDISK                     | 7.500 TB   |
| UKI-LT2-QMUL_SCRATCHDISK                 | 50.000 TB  |
| UKI-NORTHGRID-LANCS-HEP-CEPH_SCRATCHDISK | 50.000 TB  |
| UKI-NORTHGRID-LIV-HEP_SCRATCHDISK        | 13.194 TB  |
| UKI-NORTHGRID-MAN-HEP-CEPH_SCRATCHDISK   | 49.478 TB  |
| UKI-NORTHGRID-MAN-HEP_SCRATCHDISK        | 49.478 TB  |
| UKI-SCOTGRID-GLASGOW-CEPH_SCRATCHDISK    | 50.000 TB  |
| UKI-SOUTHGRID-RALPP_SCRATCHDISK          | 7.500 TB   |
| UNI-BONN_LOCALGROUPDISK                  | 10.000 TB  |
| UNI-FREIBURG_LOCALGROUPDISK              | 475.000 TB |
| UNI-FREIBURG_SCRATCHDISK                 | 18.000 TB  |
| WUPPERTALPROD_LOCALGROUPDISK             | 1.000 TB   |
| WUPPERTALPROD_SCRATCHDISK                | 19.792 TB  |
+------------------------------------------+------------+
+------------------+------------+
| RSE EXPRESSION   | LIMIT      |
|------------------+------------|
| type=SCRATCHDISK | 200.000 TB |
+------------------+------------+
```